Add community report 2025/Q1 (#14)
dot-pal/pal-homepage: docs/06_community_reports/97_2025_Q1.md

|------|-------------------------------------|-----------|-----------------|--------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 3058 | Coinfabrik Scout 50% of MS2 and MS3 | 5,205 DOT | Static Analysis | [Github](https://github.com/CoinFabrik/scout-audit/) | [offer](/tooling/24-coinfabrik-scout-offer.pdf), [milestones](/tooling/24-coinfabric-scout-milestones.xlsx) |
| 3379 | Coinfabrik Scout 50% of MS3 and MS4 | 7,066 DOT | Static Analysis | [Github](https://github.com/CoinFabrik/scout-audit/) | [offer](/tooling/24-coinfabrik-scout-offer.pdf), [milestones](/tooling/24-coinfabric-scout-milestones.xlsx) |
| 3654 | Coinfabrik Scout 50% of MS4         | 4,246 DOT | Static Analysis | [Github](https://github.com/CoinFabrik/scout-audit/) | [offer](/tooling/24-coinfabrik-scout-offer.pdf), [milestones](/tooling/24-coinfabric-scout-milestones.xlsx) |

### Bug bounties
| ID   | Bug bounty              | Co-funded  |
|------|-------------------------|------------|
| 3624 | acala-immunefi-bb-2503  | 3,304 DOT  |

### Other

> The PAL curators have received in total 9,970 DOT as remuneration for their work. This corresponds to $3,000 per curator per month, using the 30d EMA DOT price.

Reviewer: ```suggestion
The PAL curators have received a total of 9,970 DOT as remuneration for their work. This corresponds to $3,000 per curator per month, using the 7d EMA DOT price.
```

---
title: 2025 / Q1

> description: Report for the months July - December 2024

Reviewer: ```suggestion
description: Report for the months January - March 2025
```

slug: 2025-Q1
---

# PAL Community Report 2025 / Q1
Date: 22.05.2025

This Community report summarizes the activity of the Polkadot Assurance Legion (PAL) during the period January-March 2025.

## PAL

> The Polkadot Assurance Legion (PAL) is a community-driven initiative that aims to make Polkadot a safer and more attractive place for both builders and users by allocating funds from the Polkadot Treasury (bounty #22) to advance security in Rust / Polkadot SDK.

Reviewer: ```suggestion
The Polkadot Assurance Legion (PAL) is a community-driven initiative that aims to make Polkadot a safer and more attractive place for both builders and users by allocating funds from the Polkadot Treasury ([bounty #22](https://polkadot.subscan.io/bounty/22)) to advance security in Rust / Polkadot SDK.
```

PAL provides funding for audits (Rust and Solidity), security tooling, bug bounties, and common-good security initiatives. Find out more about the various funding opportunities on [PAL’s homepage](https://dotpal.io).

## Summary
In the period January-March 2025, PAL has provided funding for 2 parachain audits, 1 security tool, and 1 bug bounty.

The conducted audits helped secure **(TBA)** lines of code (LOC) and identify a total of **(TBA)** issues, **(TBA)** of which classified as High Impact.

> In the security tooling bucket, PAL provided funding for the completion of [Scout](https://github.com/CoinFabrik/scout-audit/) - a static analysis tool which enables developers to automatically identify common vulnerabilities in their Substrate runtimes (e.g. unsafe math). You can find more info about Scout on our [security tooling](/security_tooling#scout-by-coinfabrik) page.

Reviewer: ```suggestion
In the [security tooling bucket](funding/security_tooling), PAL provided funding for the completion of [Scout](https://github.com/CoinFabrik/scout-audit/) - a static analysis tool which enables developers to automatically identify common vulnerabilities in their Substrate runtimes (e.g. unsafe math). You can find more about Scout on our [security tooling](/security_tooling#scout-by-coinfabrik) page.
```

> Furthermore, PAL started a consultation round to collect feedback on its first Request for Proposals (RFP) for the development of an [ecosystem monitoring tool](https://forum.polkadot.network/t/specs-for-an-ecosystem-monitoring-tool-funded-by-pal/12160).

Reviewer: ```suggestion
Furthermore, PAL started a consultation round to collect feedback on its first Request for Proposals (RFP) for the development of an [ecosystem monitoring tool](https://forum.polkadot.network/t/specs-for-an-ecosystem-monitoring-tool-funded-by-pal/12160), denominated [RFP2501](/rfp/2501).
```

> Finally, PAL co-funded for the first time a payout under a bug bounty program.

Reviewer: ```suggestion
Finally, PAL co-funded for the first time a payout under a [bug bounty program](/funding/bug_bounty).
```

|------|-------------------------------------|-----------|-----------------|--------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 3058 | Coinfabrik Scout 50% of MS2 and MS3 | 5,205 DOT | Static Analysis | [Github](https://github.com/CoinFabrik/scout-audit/) | [offer](/tooling/24-coinfabrik-scout-offer.pdf), [milestones](/tooling/24-coinfabric-scout-milestones.xlsx) |
| 3379 | Coinfabrik Scout 50% of MS3 and MS4 | 7,066 DOT | Static Analysis | [Github](https://github.com/CoinFabrik/scout-audit/) | [offer](/tooling/24-coinfabrik-scout-offer.pdf), [milestones](/tooling/24-coinfabric-scout-milestones.xlsx) |
| 3654 | Coinfabrik Scout 50% of MS4         | 4,246 DOT | Static Analysis | [Github](https://github.com/CoinFabrik/scout-audit/) | [offer](/tooling/24-coinfabrik-scout-offer.pdf), [milestones](/tooling/24-coinfabric-scout-milestones.xlsx) |

### Bug bounties
| ID   | Bug bounty              | Co-funded  |
|------|-------------------------|------------|
| 3624 | acala-immunefi-bb-2503  | 3,304 DOT  |

### Other

> The PAL curators have received in total 9,970 DOT as remuneration for their work. This corresponds to $3,000 per curator per month, using the 30d EMA DOT price.

Reviewer: Do we want to mention that we are moving to 7d EMA?
